cmd/main.go

```diff
@@ -153,10 +168,18 @@ func main() {
 				if err != nil {
 					log.Errorf("error renewing auth: %s", err)
 				}
+				if err == vault.ErrPermissionDenied {
```

Comment:
Is there a reason we don't put it here: https://github.com/uswitch/vault-creds/pull/12/files#diff-09487838846606ad96d627de95c1cb09R51 ?

Reply:
we need to cause the cleanup function to trigger which needs variables in the main.go

Reply:
Right.